docs/api/QUICKSTART.md

```diff
@@ -193,14 +192,14 @@ Create a new source file, `~/code/uvisor-example/source/secure_box.cpp`. We will
 /* ~/code/uvisor-example/source/secure_box.cpp */
 
 #include "mbed.h"
-#include "rtos.h"
 #include "uvisor-lib/uvisor-lib.h"
 
 /* Private static memory for the secure box */
 typedef struct {
     uint32_t * buffer;          /* Static private memory, pointing to dynamically allocated private memory */
     uint32_t counter;           /* Static private memory */
     int index;                  /* Static private memory */
+    RawSerial * pc;             /* Static private memory, pointing to dynamically allocated private memory */
```

Comment:
I added this because calling `printf` from the RPC thread crashed the board with an RTX error. Using uvisor_ctx->pc->printf is fine.